.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!--
+This template is for bug reports. If you are here for another reason, please see below:
+
+1. To propose a new rule, see http://eslint.org/docs/developer-guide/contributing/new-rules
+2. To request a change, see http://eslint.org/docs/developer-guide/contributing/changes
```

Comment:
I disagree. Having too many fields and too much text (most of which people won't read) will be a barrier to getting submissions. I'd rather start with something minimal like this and make changes if/when we see problems.